docs/user/GettingStarted.md

```diff
@@ -0,0 +1,96 @@
+# Getting Started with OpenROAD Flow
+
+OpenROAD Flow is a full RTL-to-GDS flow built entirely on open-source tools.
+The project aims for automated, no-human-in-the-loop digital circuit design
+with 24-hour turnaround time.
+
+## Setup
+
+There are currently two options to get OpenROAD Flow setup:
+
+* Build from sources using Docker, [instructions here](./BuildWithDocker.md).
+* Build from sources locally, [instructions here](./BuildLocally.md).
+
+## Running a Design
+
+Sample design configurations are available in the `designs` directory.
+You can select a design using either of the following methods:
+
+1.  The flow
+    [Makefile](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/Makefile)
+    contains a list of sample design configurations at the top of the
+    file. Uncomment the respective line to select the design
+2.  Specify the design using the shell environment. For example:
+
+``` shell
+make DESIGN_CONFIG=./designs/nangate45/swerv/config.mk
+# or
+export DESIGN_CONFIG=./designs/nangate45/swerv/config.mk
+make
+```
+
+By default, the `gcd` design is selected using the
+`nangate45` platform. The resulting GDS will be available at
+`flow/results/nangate45/gcd/6_final.gds`. The flow should take only a few
+minutes to produce a GDS for this design.  We recommend implementing this
+design first to validate your flow and tool setup.
+
+### Adding a New Design
+
+To add a new design, we recommend looking at the included designs for
+examples of how to set one up.
+
+## Platforms
+
+OpenROAD-flow-scripts supports Verilog to GDS for the following open platforms:
+
+* ASAP7
+* Nangate45 / FreePDK45
+* SKY130
+
+These platforms have a permissive license which allows us to
+redistribute the PDK and OpenROAD platform-specific files. The platform
+files and license(s) are located in `platforms/{platform}`.
+
+OpenROAD-flow-scripts also supports the following commercial platforms:
+
+* GF12
+* TSMC65LP
+
```

Comment:
@maliberty do we need to add intel22 here?

Reply:
I think so but @tspyrou can confirm.

Reply:
Removed the list as they are restricted words, thus was causing security check to fail. We can revisit this later.

Reply:
We white list
    r"^(tools/OpenROAD/)?docs/user/GettingStarted.rst$",
Do we need to change that to .md?

Reply:
also     r"^(tools/OpenROAD/)?docs/index.rst$",

Reply:
Yes, that would solve the issue as well.